src/main/java/org/kainos/ea/resources/AuthController.java

```diff
@@ -0,0 +1,57 @@
+package org.kainos.ea.resources;
+
+import io.swagger.annotations.Api;
+import org.kainos.ea.api.AuthService;
+import org.kainos.ea.cli.LoginRequest;
+import org.kainos.ea.db.AuthDAO;
+import org.kainos.ea.client.DatabaseConnectionFailedException;
+import org.kainos.ea.client.JWTCouldNotBeCreatedException;
+import org.kainos.ea.client.CouldNotGeneratePasswordHashException;
+import org.kainos.ea.client.CouldNotFindUserAccountException;
+import org.kainos.ea.client.LoginDetailsAreNotInCorrectFormatException;
+import org.kainos.ea.client.InvalidLoginAttemptException;
+import org.kainos.ea.validator.AuthValidator;
+
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Api("Authentication Controller")
+@Path("/api/auth")
+public class AuthController {
+
+  private final AuthService authService = new AuthService();
```

Comment:
Your controller should have these dependencies passed to it

Reply:
Fixed Now! :(